io.github.ungoogled_software.ungoogled_chromium.yaml

```diff
@@ -0,0 +1,175 @@
+app-id: io.github.ungoogled_software.ungoogled_chromium
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+base: org.chromium.Chromium.BaseApp
+base-version: '23.08'
+command: chromium
+finish-args:
+  - --require-version=1.8.2
+  - --filesystem=home
+  - --filesystem=/run/.heim_org.h5l.kcm-socket
+  - --device=all
+  - --env=LD_LIBRARY_PATH=/app/chromium/nonfree-codecs/lib
+  - --share=ipc
+  - --share=network
+  - --socket=cups
+  - --socket=pcsc # FIDO2
+  - --socket=pulseaudio
+  - --socket=x11
+  - --socket=wayland
+  - --system-talk-name=org.bluez
+  - --system-talk-name=org.freedesktop.Avahi
+  - --system-talk-name=org.freedesktop.UPower
+  - --talk-name=com.canonical.AppMenu.Registrar
+  - --talk-name=org.freedesktop.FileManager1
+  - --talk-name=org.freedesktop.Notifications
+  - --talk-name=org.freedesktop.ScreenSaver
+  - --talk-name=org.freedesktop.secrets
+  - --talk-name=org.kde.kwalletd5
+  - --talk-name=org.kde.kwalletd6
+  - --talk-name=org.gnome.SessionManager
+  - --own-name=org.mpris.MediaPlayer2.chromium.*
+  - --persist=.pki # https://github.com/flathub/com.github.Eloston.UngoogledChromium/issues/34
+
+add-extensions:
+  io.github.ungoogled_software.ungoogled_chromium.Codecs:
+    directory: chromium/nonfree-codecs
+    add-ld-path: lib
+    bundle: true
+    autodelete: true
+
+  io.github.ungoogled_software.ungoogled_chromium.NativeMessagingHost:
+    version: '1'
+    directory: chromium/native-messaging-hosts
+    merge-dirs: native-messaging-hosts
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+
+  io.github.ungoogled_software.ungoogled_chromium.Extension:
+    version: '1'
+    directory: chromium/extensions
+    merge-dirs: extensions;native-messaging-hosts;policies/managed;policies/recommended
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+
+  io.github.ungoogled_software.ungoogled_chromium.Policy:
+    version: '1'
+    directory: chromium/policies
+    merge-dirs: policies/managed;policies/recommended
+    subdirectories: true
+    no-autodownload: true
+    autodelete: true
+
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.llvm17
+  - org.freedesktop.Sdk.Extension.node18
+  - org.freedesktop.Sdk.Extension.openjdk21
+  - org.freedesktop.Sdk.Extension.rust-nightly
```

Comment:
Why do you need rust nightly?

Reply:
Chromium requires Rust nightly, alternatively I could use the x86 prebuilts from Google but then we'd need to compile Rust on ARM as well as they don't make the prebuilts available for ARM. That's what we're doing already with Clang.

Reply:
No, it's ok if they need it. Ideally they shouldn't depend on nightly stuff.